pyvista/core/filters/data_set.py

```diff
@@ -1160,9 +1156,10 @@ def threshold_percent(
 
         Parameters
         ----------
-        percent : float or tuple(float), optional
-            The percentage (0,1) to threshold. If value is out of 0 to 1 range,
-            then it will be divided by 100 and checked to be in that range.
+        percent : float | sequence[float], optional
+            The percentage in the range ``(0, 1)`` to threshold. If value is
+            out of 0 to 1 range, then it will be divided by 100 and checked to
+            be in that range.
```

Comment:
This is a confusing API with a confusing description. I assumed this only takes percentages, which it does until you try to threshold by 0.5%. The way this is related to this PR is that we should at least try to make

> The percentage in the range ``(0, 1)`` to threshold.

less confusing. Not sure if calling it "[ratio] in the range" would be clearer/correct, for instance.

I'd consider deprecating the ratio feature sometime, potentially with the addition of a dedicated "threshold by ratio/proportion/etc." filter.

Reply:
https://github.com/pyvista/pyvista/issues/4182